docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
+SPHINXPROJ    = squid-py
```

Comment:
Is this needed?

Reply:
I'd say yes. In Fedora, at least F28, the _sphinx-build_ command is _sphinx-build-3_ , so having an easy override is handy.